ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/clock-timezone.yml

```diff
@@ -0,0 +1,2 @@
+clock:
+  timezone: GMT
```

Comment:
Missing empty new line at the End of File

Reply:
does the newline in the yaml have a purpose, or is it just cosmetical? just for my understanding